src/Selenium2Driver.php

```diff
@@ -656,12 +656,18 @@ public function setValue($xpath, $value)
         $value = strval($value);
 
         if (in_array($elementName, array('input', 'textarea'))) {
-            $existingValueLength = strlen($element->attribute('value'));
+            $deleteSequence = Key::CONTROL . 'a' . KEY::NULL_KEY . Key::COMMAND . 'a' . KEY::NULL_KEY;
```

Comment:
@aik099, yes pressing both keys causes "a" to be typed in Chrome, I'll look into OS detection.